src/web/index.jsp

```diff
@@ -69,9 +69,7 @@
 
 <%! long lastRSSFetch = 0;
     SyndFeed lastBlogFeed = null;
-    SyndFeed lastReleaseFeed = null;
     String blogFeedRSS = "https://community.igniterealtime.org/blogs/ignite/feeds/posts";
-    String releaseFeedRSS = "https://community.igniterealtime.org/community/feeds/messages?community=2017";
 
```

Comment:
I'm not a dev, so it was the fastest thing to change :) One can add an option here.